src/world/Server/Script/CreatureAIScript.cpp

```diff
@@ -349,6 +391,26 @@ uint8_t CreatureAIScript::getAIAgent()
     return _creature->getAIInterface()->getCurrentAgent();
 }
 
+void CreatureAIScript::setReactState(ReactStates st)
+{
+    getCreature()->getAIInterface()->setReactState(st);
```

Comment:
you can use _creature instead of  getCreature() (less function calls for own class members)

Reply:
indeed true but i was not sure here what i should do thank you